build-api.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Generate files for examples-api to `dist` folder
+ *
+ * API ENDPOINTS
+ *
+ *     Get a list of example names in zeit/now-examples
+ *     /_api/list.json
+ *
+ *     Get detailed list of all examples
+ *     /_api/manifest.json
+ *
+ *     Download example files by name (e.g. "apollo")
+ *     /_api/download/apollo.tar.gz
+ */
+
+const debuglog = require('util').debuglog('API_BUILDER');
+const { lstatSync, readdirSync, createWriteStream } = require('fs');
+const { join } = require('path');
+const fse = require('fs-extra');
+const tar = require('tar-fs');
+
+main();
```

Comment:
```suggestion
main()
  .then(() => process.exit(0))
  .catch((err) => {
    console.error(err);
    process.exit(1);
  });
```

Just to make sure it actually exits, sometimes a different process will keep running after `main` exits, though, it is probably unlikely in this case.